frontend/apps/reader/modules/readerfrontlight.lua

```diff
@@ -75,11 +196,22 @@ function ReaderFrontLight:onShowIntensity()
     end
     UIManager:show(Notification:new{
         text = new_text,
-        timeout = 2,
+        timeout = 1,
     })
     return true
 end
 
+function ReaderFrontLight:onShowWarmth(value)
+    local powerd = Device:getPowerDevice()
+    if powerd.fl_warmth ~= nil then
+        UIManager:show(Notification:new{
+            text = T(_("Warmth is set to %1."), powerd.fl_warmth),
```

Comment:
```suggestion
            text = T(_("Warmth set to %1."), powerd.fl_warmth),
```

Just from the shorter is better perspective for popups, as long as it's properly grammatical and all.